src/lavinmq/replication/server.cr

```diff
@@ -98,6 +98,19 @@ module LavinMQ
         end
       end
 
+      def wait_for_max_lag
+        until @followers.size >= Config.instance.min_followers
+          select
+          when @wait_for_followers.receive
+          when timeout(1.seconds)
+          end
```

Comment:
I think you can replace this with, or can `@follower.size` or `Config.instance.min_followers` change value while waiting without sending to `@wait_for_followers`?
```suggestion
@wait_for_followers.receive
```

Reply:
Yes, currently @follower.size can change without sending to the channel. We recieve on this channel when a follower connects and starts reading.. but other followers can disconnect when ever